pkg/analysis/maxlength/analyzer_test.go

```diff
@@ -0,0 +1,14 @@
+package maxlength_test
+
+import (
+	"testing"
+
+	"github.com/JoelSpeed/kal/pkg/analysis/maxlength"
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestDefaultConfiguration(t *testing.T) {
```

Comment:
Nit: Test name says this tests the default configuration, but this linter doesn't seem to be configurable. Might be a touch misleading.

Reply:
Fair, wonder if in the future we might want to have any config here 🤔 Possibly, but, we can always rename this back later